spec/features/teacher_sign_in_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe Teacher, type: :feature do
+  def sign_in(teacher)
+    fill_in 'teacher_email', with: teacher.email
+    fill_in 'teacher_password', with: teacher.password
+    click_on 'Log in'
+  end
+  sleep(3)
+  describe 'signs in' do
+    it 'successfully' do
+      Capybara.current_driver = :selenium_chrome
+      teacher = create(:teacher, attributes_for(:teacher))
```

Comment:
it can just be `create(:teacher)`, if you pass no attributes FactoryBot will create them for you :)

Reply:
ping @mtrojanowska

Reply:
Check :)

Reply:
check in all places apart from this one :) please also fix here

Reply:
Yeah. Stupid me. So sorry :(
Check